canopen/network.py

```diff
@@ -279,6 +279,17 @@ def __len__(self) -> int:
         return len(self.nodes)
 
 
+class _DummyNetwork(Network):
```

Comment:
I'm not sure about the naming. I'd prefer if the naming more closely reflected it's purpose:

- `_NetworkPlaceholder`
- `_UninitializedNetwork`
- `_EmptyNetwork`